src/Aspire.Hosting.Azure.Storage/AzureBlobStorageResource.cs

```diff
@@ -26,7 +29,25 @@ public class AzureBlobStorageResource(string name, AzureStorageResource storage)
     public ReferenceExpression ConnectionStringExpression =>
        Parent.GetBlobConnectionString();
 
-    void IResourceWithAzureFunctionsConfig.ApplyAzureFunctionsConfiguration(IDictionary<string, object> target, string connectionName)
+    internal ReferenceExpression GetConnectionString(string? blobContainerName)
+    {
+        if (string.IsNullOrEmpty(blobContainerName))
+        {
+            return ConnectionStringExpression;
+        }
+
+        var builder = new ReferenceExpressionBuilder();
+        builder.Append($"{ConnectionStringExpression}");
+
+        if (!string.IsNullOrEmpty(blobContainerName))
+        {
+            builder.Append($"/{blobContainerName}");
```

Comment:
Can we always just append this at the end? Even for local/emulator connection strings?

Reply:
We're always appending it except when it's empty (which it shouldn't be). Or is there something I'm missing?

Reply:
This code is assuming that the code in 

https://github.com/dotnet/aspire/pull/9008/files#diff-8609a05ae8177771a0d588a2703d7fcf8ba85134b4d39670154e67816dddb328R15

will always be at the end, and it doesn't have a trailing `;`.

Reply:
Eventually, I figured what I've done :) Reverted all of these.
Thanks